sharding-core/sharding-core-merge/src/main/java/org/apache/shardingsphere/core/merge/dal/show/ShowDatabasesMergedResult.java

```diff
@@ -30,22 +31,42 @@
 import java.sql.SQLXML;
 import java.util.Collections;
 import java.util.List;
+import org.apache.shardingsphere.core.rule.ShardingRule;
 
 /**
  * Merged result for show databases.
  *
  * @author chenqingyang
  */
-@RequiredArgsConstructor
 public final class ShowDatabasesMergedResult extends LocalMergedResultAdapter implements MergedResult {
     
     private final List<String> schemas;
     
     private int currentIndex;
-    
+
     public ShowDatabasesMergedResult() {
         this(Collections.singletonList(ShardingConstant.LOGIC_SCHEMA_NAME));
     }
+
+    //for proxy backend, just return logic database
```

Comment:
please remove all inline comments, we permit javado todo and fixme comment only

Reply:
Done